transit_odp/publish/templates/publish/seasonal_services/edit_date.html

```diff
@@ -0,0 +1,60 @@
+{% extends "publish/base.html" %}
+
+{% load i18n %}
+{% load crispy_forms_tags %}
+{% load breadcrumbs %}
+
+{% block title_tag %}
+    {% if form.errors %}{% trans 'Error: ' %}{% endif %}
+    {% if title_tag_text %}
+        {{ title_tag_text }}
+    {% else %}
+        {{ block.super }}
+    {% endif %}
+{% endblock %}
+
+
+{% block breadcrumb.content %}
+    {{ block.super }}
+    {% if user.is_agent_user %}
+        {% with "Review "|add:organisation.name|title|add:" Timetables Data" as heading %}
+            {% breadcrumb_url heading 'feed-list' pk1=pk1 host hosts.publish %}
+        {% endwith %}
+    {% else %}
+        {% breadcrumb_url "Review My Timetables Data" 'feed-list' pk1=pk1 host hosts.publish %}
+    {% endif %}
+    {% breadcrumb_url "Manage seasonal services" "seasonal-service" pk1=pk1 host hosts.publish %}
+    {% breadcrumb "Edit dates" %}
+{% endblock %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-full">
+            <h1 class="govuk-heading-xl">{% trans 'Seasonal service operating dates' %}</h1>
+        </div>
+    </div>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-one-half">
+            <div class="heading-container">
+                <h2 class="govuk-heading-s govuk-!-margin-bottom-2">{% trans "PSV Licence Number" %}</h2>
+            </div>
+            <p class="govuk-body">
+                {{ form.instance.licence.number }}
+            </p>
+            
+            <div class="heading-container">
+                <h2 class="govuk-heading-s govuk-!-margin-bottom-2">{% trans "Service code" %}</h2>
+            </div>
+            <p class="govuk-body">
+                {{ form.instance.licence.number }}/{{ form.instance.registration_code }}
+            </p>
+            
+            <hr class="govuk-section-break govuk-section-break--xs govuk-section-break--visible govuk-!-margin-bottom-8 govuk-!-margin-top-8">
+            
+            <form class="seasonal-service" method="post" enctype="multipart/form-data" novalidate>
+                {% crispy form %}
```

Comment:
The real problem here is that this will be where the form gets rendered. So the errors which appear at the top of the form will appear here.

The real useful stuff here is from line 41 to 52, Id make this a seasonal service snippet and use render_to_string and HTML in the layout. You can find an example of this [here](https://github.com/department-for-transport-BODS/bods/blob/dev/transit_odp/publish/forms/seasonal_services.py#L135)

Reply:
Absolutely Correct!! Thanks

Reply:
Now I got you!! 😉